styles.css

```diff
@@ -179,7 +177,7 @@ p {
   padding-left: 20px;
 }
 
-#popupmobile{
+#popupmobile {
```

Comment:
- [ ] In the mobile version I think just you need to align the heading with the technologies to be flex-start so kindly fix it 

- [ ] I think it would be great if you add font weight to the heading here since it has more font weight in the Figma design 😊💯🔥🙏🏻
- Your design
![image](https://user-images.githubusercontent.com/58553711/188890066-bbef28a1-4483-47c1-aa20-32661972f659.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/188890189-82e6b900-a7aa-4b65-b0e1-81e4f98c1f55.png)

Reply:
ok i put flex start in this section, thank you very much